server/utils/placementRequests/table.ts

```diff
@@ -0,0 +1,59 @@
+import { add } from 'date-fns'
+import { PlacementCriteria, PlacementRequest } from '../../@types/shared'
+import { TableCell, TableRow } from '../../@types/ui'
+import { DateFormats } from '../dateUtils'
+import { sentenceCase } from '../utils'
+
+export const DIFFERENCE_IN_DAYS_BETWEEN_DUE_DATE_AND_ARRIVAL_DATE = 7
+
+export const tableRows = (placementRequests: Array<PlacementRequest>): Array<TableRow> => {
+  return placementRequests.map((placementRequest: PlacementRequest) => {
+    return [
+      nameCell(placementRequest),
+      crnCell(placementRequest),
+      dueDateCell(placementRequest, DIFFERENCE_IN_DAYS_BETWEEN_DUE_DATE_AND_ARRIVAL_DATE),
+      expectedArrivalDateCell(placementRequest),
+      locationCell(placementRequest),
+      essentialPlacementCriteriaCell(placementRequest),
+      desirablePlacementCriteriaCell(placementRequest),
+    ]
+  })
+}
+
+const nameCell = (placementRequest: PlacementRequest): TableCell => ({ text: placementRequest.person.name })
+const crnCell = (placementRequest: PlacementRequest): TableCell => ({ text: placementRequest.person.crn })
+
+export const dueDateCell = (
+  placementRequest: PlacementRequest,
+  differenceBetweenDueDateAndArrivalDate: number,
+): TableCell => {
+  const dateAsObject = DateFormats.isoToDateObj(placementRequest.expectedArrival)
+
+  return {
+    text: DateFormats.differenceInDays(
+      dateAsObject,
+      add(dateAsObject, { days: differenceBetweenDueDateAndArrivalDate }),
+    ).ui,
+  }
+}
+
+export const expectedArrivalDateCell = (placementRequest: PlacementRequest): TableCell => ({
+  text: DateFormats.isoDateToUIDate(placementRequest.expectedArrival),
+})
+
+export const essentialPlacementCriteriaCell = (placementRequest: PlacementRequest): TableCell => ({
+  html: `<ul>${placementNeedsSentenceCase(placementRequest.essentialCriteria)}</ul>`,
```

Comment:
Could we maybe remove the spacing in the list items (and maybe make the font a bit smaller?) Just looks a bit odd in the table cells.